src/Validator/Range.php

```diff
@@ -31,14 +31,21 @@ class Range extends Numeric
      */
     protected $max;
 
+    /**
+     * @var string
+     */
+    protected $format;
+
     /**
      * @param int $min
      * @param int $max
+     * @param string $format
      */
-    public function __construct($min, $max)
+    public function __construct($min, $max, $format='int32')
```

Comment:
```suggestion
    public function __construct($min, $max, $format = self::TYPE_INTEGER)
```

@lohanidamodar it will be wiser to only accept here the constant that this class already support rather than free style strings